### PR TITLE
Bump CoreDNS to GA in kubeadm

### PR DESCRIPTION
**Pull request by rajansandeep**
**What this PR does / why we need it**:
Graduates CoreDNS to GA.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/features/features.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ var InitFeatureGates = FeatureList{
 	StoreCertsInSecrets: {FeatureSpec: utilfeature.FeatureSpec{Default: false, PreRelease: utilfeature.Alpha}},
 	// We don't want to advertise this feature gate exists in v1.9 to avoid confusion as it is not yet working
 	HighAvailability:     {FeatureSpec: utilfeature.FeatureSpec{Default: false, PreRelease: utilfeature.Alpha}, MinimumVersion: v190, HiddenInHelpText: true},
-	CoreDNS:              {FeatureSpec: utilfeature.FeatureSpec{Default: false, PreRelease: utilfeature.Beta}, MinimumVersion: v190},
+	CoreDNS:              {FeatureSpec: utilfeature.FeatureSpec{Default: false, PreRelease: utilfeature.GA}, MinimumVersion: v190},
```
Comment (timothysc) [inline cmd/kubeadm/app/features/features.go]: default - true please .

**Review by timothysc — CHANGES_REQUESTED** [review]
/cc @luxas

**Comment by detiber** [comment]
I also cherry-picked this into my PR for defaulting the CoreDNS feature flag to true: https://github.com/kubernetes/kubernetes/pull/63509

**Comment by fturib** [comment]
@timothysc @detiber @luxas : after talk with @thockin, @bowei - it was agreed that CoreDNS will go GA this version 1.11 of k8s, but will be the default DNS service only after some adoption by users - which means at earlieast v1.12

So, the KEP was changed to include only the GA part : https://github.com/kubernetes/community/pull/1956

Another KEP has to ne opened for the "Default" part.


and the initial feature description was also changes was now 2 features proposed opened:

- one for GA : https://github.com/kubernetes/features/issues/427
- one for Default : https://github.com/kubernetes/features/issues/566

**Comment by neolit123** [comment]
can be closed given it was cherry picked here #63509.

**Comment by luxas** [comment]
@fturib We're enabling it by default in v1.11 anyway. That way we'll also be kind of a guinea pig (although that really shouldn't be the case for GA) and maybe catch potential issues before v1.12. It's probably better than the current kube-dns anyway.

**Comment by luxas** [comment]
Closing in favor of #63509. Thanks!

**Pull request closed without merging**